src/site/content/en/blog/multi-screen-window-placement/index.md

```diff
@@ -168,106 +171,154 @@ The new `window-placement` permission can be queried with the
+    pixelDepth: 30
+    pointerTypes: []
+    primary: false
+    scaleFactor: 2
+    top: 0
+    touchSupport: false
+    width: 1680
+  },
+  {
+    // The iPad
+    availHeight: 999
+    availLeft: 1680
+    availTop: 25
+    availWidth: 1366
+    colorDepth: 24
+    devicePixelRatio: 2
+    height: 1024
+    id: ""
+    internal: false
+    isExtended: true
+    isInternal: false
+    isPrimary: false
+    label: ""
+    left: 1680
+    onchange: null
+    orientation: ScreenOrientation {angle: 0, type: "landscape-primary", onchange: null}
+    pixelDepth: 24
+    pointerTypes: []
+    primary: false
+    scaleFactor: 2
+    top: 0
+    touchSupport: false
+  }]
+}
 */
 ```
 
-Note how the value of `left` for the iPad starts at `1680`, which is exactly the `width` of the
-built-in display. This allows me to determine exactly how the screens are arranged logically (next
-to each other, on top of each other, etc.). There is also data now for each screen to show whether
-it is an `internal` one and whether it is a `primary` one.
-Note that the built-in screen
+Information about the connected screens is available in the `screens` array. Note how the value of
+`left` for the iPad starts at `1680`, which is exactly the `width` of the built-in display. This
+allows me to determine exactly how the screens are arranged logically (next to each other, on top of
+each other, etc.). There is also data now for each screen to show whether it is an `isInternal` one
+and whether it is an `isPrimary` one. Note that the built-in screen
 [is not necessarily the primary screen](https://osxdaily.com/2010/04/27/set-the-primary-display-mac/#:~:text=Click%20on%20the%20Display%20icon,primary%20display%20for%20your%20Mac).
-Both also have an `id`, which, if
-persisted across browser sessions, allows for window arrangements to be restored.
+Both also have a `label` and an `id`, which, if persisted across browser sessions, allows for window
+arrangements to be restored. Information about the possible input methods are available in the
+`pointerTypes` array.
+
+The field `currentScreen` is a live object corresponding to the current `window.screen`. The object
+is updated on cross-screen window placements or device changes.
 
 ### The `screenschange` event
 
 The only thing missing now is a way to detect when my screen setup changes. A new event,
-`screenschange`, does exactly that: it fires whenever the screen constellation
-is modified.
-(Notice that "screens" is plural in the event name.)
-It also fires when the resolution of one of the connected screens changes or when a
-new or an existing screen is (physically or virtually in the case of Sidecar) plugged in or unplugged.
+`screenschange`, does exactly that: it fires whenever the screen constellation is modified. (Notice
+that "screens" is plural in the event name.) It also fires when the resolution of one of the
+connected screens changes or when a new or an existing screen is (physically or virtually in the
```

Comment:
`screenschange` does *not* fire when attributes of a particular screen change, only when some screens are added or removed from the `screens` array. For that, users should use the new `change` event on `Screen` (and `ScreenAdvanced`). Let us know if we can clarify https://webscreens.github.io/window-placement/#api-screens-events or the explainer, etc.

Reply:
Thanks, clarified.